From ec08df5292824c64f62eea6fcebb7c8d870c4715 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Thu, 5 Jun 2014 01:34:44 +1000
Subject: [PATCH 1/3] Fix crash when exiting LiveTV

This can occur mostly when liveTV is started using mythavtest. Should playback fail for some reasons and we exited: kaboom
---
 mythtv/libs/libmythtv/tvbrowsehelper.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/tvbrowsehelper.cpp b/mythtv/libs/libmythtv/tvbrowsehelper.cpp
index 1843d93..f33cc63 100644
--- a/mythtv/libs/libmythtv/tvbrowsehelper.cpp
+++ b/mythtv/libs/libmythtv/tvbrowsehelper.cpp
@@ -273,7 +273,7 @@ uint TVBrowseHelper::GetChanId(
 void TVBrowseHelper::GetNextProgram(
     BrowseDirection direction, InfoMap &infoMap) const
 {
-    if (!m_ctx->recorder)
+    if (!m_ctx || !m_ctx->recorder)
         return;
 
     QString title, subtitle, desc, category, endtime, callsign, iconpath;
-- 
1.7.10.2

